python/tvm/relay/__init__.py

```diff
@@ -120,3 +123,52 @@
 ModulePass = transform.ModulePass
 FunctionPass = transform.FunctionPass
 Sequential = transform.Sequential
+
+def int32(val):
+    return relay.const(val, 'int32')
+
+def type_of(sb, expr):
+    if isinstance(expr, Var):
+        return expr.type_annotation
+    else:
+        it = IncompleteType()
+        v = var("unify", it)
+        sb.let(v, expr)
+        return it
+
+# TODO(@jroesch): possibly move to their own file
```

Comment:
Indeed, the iterators probably are worth documenting further and having separately